### DISCUSSION TITLE
blog/2018/03/08/amelia-broom-huxtable/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#82] [Blog comments]
# blog/2018/03/08/amelia-broom-huxtable/

Extend broom’s tidy() and glance() to work with lists of multiply imputed regression models

https://www.andrewheiss.com/blog/2018/03/08/amelia-broom-huxtable/

**Comment by nnray** [comment]
Incredibly helpful. Created the described tables for my various models after slight adaptation. I will be sure to cite this page in my manuscript. Thank you!
